### DISCUSSION TITLE
products ,orders and other tables improvements

### DISCUSSION THREAD

**Discussion by boudo2017** [Blair2004/NexoPOS#515] [Ideas]
Hi,
It will be better to make all tables rows clickable , interactive .you can add some actions to appeared popup.
regards

![Screenshot (4)1](https://user-images.githubusercontent.com/22215705/149638487-112c9ad5-aba2-41e5-95d5-26c51819e1f8.png)

**Comment by Blair2004** [comment]
The line should be clickable what for ? Note a line might have various options. Having a line clickable for editing is therefore no appropriate. What about those who want to copy a value from the line ? They'll mistakenly trigger the click event.

Thanks for the suggestions, but that's will brings more issue than solutions.
